crates/oxc_semantic/tests/fixtures/oxc/js/declarations/using.js

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Could you add `await using` to this test?

Reply:
done, let me know if that's what you had in mind

Reply:
Thanks. Yes, that's exactly what I meant.